doc/nvim-treesitter.txt

```diff
@@ -447,4 +452,25 @@ Literal text.
 								    *hl-TSURI*
 Any URI like a link or email.
 
+==============================================================================
+MODULE-HIGHLIGHTS                            *nvim-treesitter-module-highlights*
+
+Apart from the general purpose highlights in *nvim-treesitter-highlights* ,
```

Comment:
```suggestion
Apart from the general purpose highlights in *nvim-treesitter-highlights*,
```

Reply:
space is need for the * to be recognized by the regex engine for highlighting/concealing.

Reply:
that's why we can't have nice things with regex p: